forward_engineering/helpers/validateAvroScript.js

```diff
@@ -18,6 +18,7 @@ let _;
 const validateAvroScript = (script, scriptType, logger) => {
 	_ = dependencies.lodash;
 
+	scriptType = detectScriptType(script) || scriptType;
```

Comment:
if we detected that we have regular valid avsc file - no need to parse it like schema registry